docs(ai-ready): add backport skill (#9116)
camunda/camunda-docs: .github/instructions/repo.instructions.md

- The "Next" (unreleased) docs live in `/docs/`. Versioned docs live in `/versioned_docs/version-*/`.
- Sidebar navigation is managed in `sidebars.js` (Next) and `versioned_sidebars/version-*-sidebars.json` (versioned).

> - The current released version is set in `src/versions.js` (`currentVersion`). Active maintained versions are listed in `versions.json`. Docusaurus is configured with `lastVersion: currentVersion`, so the released version is served with no URL prefix. When a new version is cut, `currentVersion` and the unprefixed version shift accordingly.

Reviewer: ```suggestion
- Version configuration: `src/versions.js` sets `currentVersion` (the default released version served without URL prefix), and `versions.json` lists all active maintained versions. Docusaurus is configured with `lastVersion: currentVersion`, so when a new version is cut, `currentVersion` and the unprefixed version shift accordingly.
```
